.circleci/config.yml

```diff
@@ -140,9 +140,16 @@ jobs:
           command: |
             # Only push to the cache when CircleCI makes the CACHIX_SIGNING_KEY
             # available (e.g. not for pull requests).
-            if [ -n "${CACHIX_SIGNING_KEY:-""}" ]; then
+            if [ -n "${CACHIX_AUTH_TOKEN:-""}" ]; then
               echo "Building and caching all derivations..."
+              cachix authtoken "$CACHIX_AUTH_TOKEN"
+
+              # Push new builds as we go
               nix-build | cachix push postgrest
+
+              # Make sure that everything, including .drv files, is pushed
+              nix-env -f default.nix -iA devtools
+              postgrest-push-cachix
```

Comment:
Does it need to call cachix twice?

Reply:
To be both thorough and fast yes - the first one pushes to cachix while we build. That misses some .drv files etc though, so we need the second pass. We could only have the second, but that would mean we waste upload bandwidth while we build

Reply:
Ah, I see. Cool!